### PR TITLE
[DX] Add GitHub issues templates

### PR DESCRIPTION
**Pull request by pharmpy-dev-123**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        For R users
+        ```r
+        packageVersion("pharmr")
+        print_pharmpy_version()
+        ```
+
+        For Python users
+        ```sh
+        pharmpy --version
```
Comment (rikardn) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:22]: There is `pharmpy.__version__`  in python

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Please describe what you are trying to achieve and what went wrong.
+        Please include the stack trace of the error (error message) you are
```
Comment (rikardn) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:10]: Perhaps put `error message' first and `including the stack trace if available` or similar in parens. To simplify the message for less tech savvy.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        witnessing.
+        If possible, please specify the version of `Pharmpy` you are running:
+
+        For R users
+        ```r
+        packageVersion("pharmr")
+        print_pharmpy_version()
```
Comment (rikardn) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:17]: Is this needed with the latest pharmr? @stellabelin added this to all errors

**Review by rikardn — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 3 above)
Comment (pharmpy-dev-123) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:17]: Oh! Super cool. I'll remove that.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 2 above)
Comment (pharmpy-dev-123) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:10]: Indeed!

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 1 above)
Comment (pharmpy-dev-123) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:22]: OK.

Comment (pharmpy-dev-123) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:21]: Done.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 2 above)
Comment (pharmpy-dev-123) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Done.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 3 above)
Comment (pharmpy-dev-123) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Done.

**Pull request merged**